uniprot-search/src/main/resources/literature/search-fields.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "field": "id",
+    "fieldValidRegex": "^[0-9]+$",
+    "sortField": "id_sort"
+  },
+  {
+    "field": "title"
```

Comment:
title_sort field is missing

Reply:
gosh, can't believe I missed this -- thank you. repetitive work causes probs!